tests/models/test_product_live_time.py

```diff
@@ -0,0 +1,71 @@
+"""Unit tests for ProductLiveTime."""
+
+import datetime
+
+from tests.testutils import db
+from timed_release.models import product_live_time
+
+
+def test_to_dict():
+    """Test that the to dict function only returns the expected properties."""
+    test_timed_release = product_live_time.ProductLiveTime(
+        product_id='12',
+        time_of_day_product='00:00:01',
+        time_zone='GMT',
+        store_id=1)
+    expected_dict = {
+        'product_id': test_timed_release.product_id,
+        'time_of_day_product': test_timed_release.time_of_day_product,
+        'time_zone': test_timed_release.time_zone,
+        'store_id': test_timed_release.store_id}
+
+    assert test_timed_release.to_dict() == expected_dict
+
+
+@db.test_schema
+def test_get_product_live_time_details_success():
+    """Test to get product live time by product id."""
+    product_id = '12'
+    data = {
+        'product_id': product_id,
+        'time_of_day_product': datetime.time(20, 30, 00),
+        'time_zone': 'GMT',
+        'store_id': 1}
+    expected_response = {
```

Comment:
because we cannot assert 'time_of_day_product': datetime.time(20, 30, 00), this value